src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -274,18 +274,18 @@ export const ShippingRoute: FC<ShippingProps> = props => {
       hasError: invalidPhoneNumber,
     } = validatePhoneNumber(phoneNumber)
 
+    if (invalidPhoneNumber) {
+      setPhoneNumberError(phoneNumberError!)
+      setPhoneNumberTouched(true)
+      !invalidAddress && jumpTo("phoneNumberTop", { behavior: "smooth" })
```

Comment:
This works, but the logic may be easier to read if instead of adding a `!invalidAddress &&` here, you use an if-else statement to decide where to jump to:

```
if (invalidAddress) {
  setAddressErrors(addressErrors!)
  setAddressTouched(touchedAddress)
  jumpTo("deliveryAddressTop", { behavior: "smooth" })
} else if (invalidPhoneNumber) {
  setPhoneNumberError(phoneNumberError!)
  setPhoneNumberTouched(true)
  jumpTo("phoneNumberTop", { behavior: "smooth" })
}
```

Reply:
Oh, never mind. Then the state hooks won't all get updated.